.github/workflows/r_package.yml

```diff
@@ -312,9 +310,7 @@ jobs:
       - name: Install packages and run tests
         shell: bash
         run: |
-          R_LIBS="c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl')"
-          Rscript \
-            -e "install.packages(${R_LIBS}, repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())"
+          Rscript ./.ci/install-r-deps.R --build --test
```

Comment:
I don't see `testthat` in `master` here...

https://github.com/microsoft/LightGBM/blob/c9de57b09cf9293213fa41b423e2409ebd9ae5c1/.github/workflows/r_package.yml#L340-L341

Reply:
Ah you are right. Adding `--test` here was to pull in `{RhpcBLASctl}`. I will rework this.

Reply:
I modified this in https://github.com/microsoft/LightGBM/pull/6808/commits/dae117f3d94d6c49873e6759b4da1b0a8bd7ea0e#diff-3e528906c029b6309d1c7491be6e3dbeaca567e4dde8a6257e3efa1356024827, to this:

```shell
Rscript ./.ci/install-r-deps.R --build --test --exclude=testthat
```

`{testthat}` is installed later:

https://github.com/microsoft/LightGBM/blob/81922a7e8fd62a28c5b3035f69ad6d6285f7ac69/.github/workflows/r_package.yml#L340-L341